src/figuresClasses.ts

```diff
@@ -1,11 +1,75 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: () => number;
```

Comment:
The `getArea` method in the `Figure` interface should not accept any parameters according to the checklist. Remove the parentheses to indicate that no parameters are expected.